### PR TITLE
Exoplanet crashed fighter

### PR DESCRIPTION
**Pull request by gilvian**
I hope this time this is the correct BoH-Bay.

Also adds a crashed fighter onto exoplanets I guess.
Spoiler for people that want to see it in game:
<details>
  
image
 
</details>

:cl: Gibong
rscadd: New exoplanet ruin (Crashed Sol Fighter)
/:cl:

### CODE REVIEW

File: maps/random_ruins/exoplanet_ruins/fighter_exoplanet/fighter_exoplanet.dm [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/datum/map_template/ruin/exoplanet/fighter_exoplanet
+	name = "Crashed SolGov Fighter"
+	id = "fighter_exoplanet"
+	description = "Crashed SolGov Fighter with a HE missile"
+	suffixes = list("fighter_exoplanet/fighter_exoplanet.dmm")
+	cost = 1
+	template_flags = TEMPLATE_FLAG_NO_RUINS | TEMPLATE_FLAG_CLEAR_CONTENTS
+	ruin_tags = RUIN_ALIEN
```
Comment (ghost) [inline maps/random_ruins/exoplanet_ruins/fighter_exoplanet/fighter_exoplanet.dm]:
```suggestion
	ruin_tags = RUIN_HUMAN | RUIN_WRECK
```

**Review by ghost — COMMENTED** [review]
Generally it's bad practice to use items that are specific to maps, like the dog tags or seal decal. This, however, is negligible since there will probably never be a situation in which this map is loaded but the Dagon is not.

**Comment by Kaksisilma** [comment]
> Generally it's bad practice to use items that are specific to maps, like the dog tags or seal decal. This, however, is negligible since there will probably never be a situation in which this map is loaded but the Dagon is not.

It often creates runtimes, but this is not the case here so *shrug

**Comment by Kaksisilma** [comment]
Please add it to Baystation.DME

Capture

You must add the away site to this file, preferably below this big bit with a lot of them on pic related.

**Comment by N8-Toe** [comment]
its a good first PR! but before we can get it merged you'll need to fix the issues pointed out by the other maintainers and we should be good to go. The map itself looks pretty good.

**Comment by gilvian** [comment]
wait i think i fucked somethign

**Comment by gilvian** [comment]
how

**Comment by N8-Toe** [comment]
it appears you made a mistake in your last commit and took it off the baystation.DME

**Comment by gilvian** [comment]
mind someone pressing the button?

**Comment by N8-Toe** [comment]
right its out of the baystation.dme again, it has to stay in there or it'll not show up in game

**Comment by gilvian** [comment]
i will cry

**Comment by gilvian** [comment]
You know what, fuck this i know what i did wrong I'm gonna go sacrifice a goat or seven to fix this

**Pull request closed without merging**